imap_processing/tests/ena_maps/test_ena_maps.py

```diff
@@ -178,9 +178,16 @@ def test_project_pset_values_to_map_push_method(self, mock_frame_transform_az_el
         assert rectangular_map.data_dict != {}
 
         # Check that the map has the same values as the PSETs, summed
-        simple_summed_pset_counts = np.sum(
-            [pset["counts"].values for pset in self.l1c_pset_products], axis=0
-        ).reshape(rectangular_map.data_dict["counts"].shape)
+        simple_summed_pset_counts = np.zeros(rectangular_map.data_dict["counts"].shape)
+        for pset in self.l1c_pset_products:
+            reshaped_pset_counts = pset["counts"].squeeze("epoch")
+            # transpose to put az, el first and then reshape to the map's counts shape
+            reshaped_pset_counts = reshaped_pset_counts.transpose(
+                "energy_bin_center",
+                "azimuth_bin_center",
+                "elevation_bin_center",
+            ).data.reshape(rectangular_map.data_dict["counts"].shape)
```

Comment:
You are putting az, el at the end now. Do you need to transpose here, or are you able to just directly add these now because you are stating that everything is coming in and going out with the spatial axes last?

Reply:
You're right, this is no longer necessary - removed!